content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -60,7 +60,7 @@ different Kubernetes components.
 | `CSIDriverRegistry` | `true` | Beta | 1.14 | |
 | `CSIInlineVolume` | `false` | Alpha | 1.15 | - |
 | `CSIMigration` | `false` | Alpha | 1.14 | |
-| `CSIMigrationAWS` | `false` | Alpha | 1.14 | |
+| `CSIMigrationAWS` | `false` | Beta | 1.16 | |
```

Comment:
Please do not remove the `Alpha` entry. It is useful to have from which version the feature gates is introduced and in which it is upgraded.

Reply:
@leakingtapan would you be able to revise this PR to leave in the mention of the alpha feature?

Reply:
done